### DISCUSSION TITLE
Home assistant e scheda relè 16 canali con porta RJ45

### DISCUSSION THREAD

**Discussion by cl-7k-01** [indomus/forum#297] [Hardware & Firmware]
Buongiorno a tutti,
uso da ormai diversi mesi prodotti della linea shelly (2.5 e 1 tutti con firmware orginale) e un sonoff mini r3 (tasmota) abbinati con successo ad HASSIO su Raspberry Pi4.
Tutto viaggia via Wifi su un sistema di router mesh sparso per la casa. 
Ogni giorno che passa però mi viene in mente di domotizzare qualcosa di "nuovo", anzi, qualcosa che ancora domotizzato non è.
La via più agevole resta sempre usare il WiFi ma vorrei poter capire se è possibile percorrere anche la strada "filare".
A tal proposito mi chiedo: una scheda a relè come [questa](https://it.aliexpress.com/item/1005001676767681.html?spm=a2g0o.productlist.0.0.67957947ysnlMb&algo_pvid=a5e2e1e7-cac3-48ed-8e0b-ef8935fe3c96&algo_expid=a5e2e1e7-cac3-48ed-8e0b-ef8935fe3c96-17&btsid=0b0a187916207233899114312e7769&ws_ab_test=searchweb0_0,searchweb201602_,searchweb201603_) è interfacciabile con Home Assistant? Se si, come?
Grazie a tutti per l'attenzione.

**Comment by maatx** [comment]
occhio che non sempre va bene con questi relè cinesi.
Io ne ho preso uno simile con 8 relè fotoaccoppiati controllabili con nodemcu. La metà non funzionano...

**Reply by cl-7k-01** [reply]
Si, in effetti l'affidabilità dei componenti è un aspetto da non sottovalutare.
Mi ero lusingato di poter comprare lo Shelly 4 PRO PLUS ma è non arriva più.. Allora ecco che mi guardavo in giro in cerca di qualcosa che potesse vagamente svolgere lo stesso ruolo.
